packages/fiori3/src/components/FilterBar/demo.stories.tsx

```diff
@@ -8,6 +8,7 @@ import { PlacementType } from '../../lib/PlacementType';
 import { TitleLevel } from '../../lib/TitleLevel';
 import { FilterType } from '../../lib/FilterType';
 import { Switch } from '../../lib/Switch';
+import { Input } from '../../webComponents/Input';
```

Comment:
Please import from lib folder
```suggestion
import { Input } from '../../lib/Input';
```